raiden/network/transport/matrix/transport.py

```diff
@@ -965,18 +966,24 @@ def _send_raw(self, receiver_address: Address, data: str) -> None:
             # `whitelist`, and the room can be used by a another thread.
             # - The room should be created by the partner, and this node is waiting
             # on it.
+            # - No user for the requested address is online
             #
             # This is not a problem since the messages are retried regularly.
-            self.log.error("No room for receiver", receiver=to_checksum_address(receiver_address))
+            self.log.warning(
+                "No room for receiver", receiver=to_checksum_address(receiver_address)
+            )
 
-    def _get_room_for_address(self, address: Address) -> Optional[Room]:
+    def _get_room_for_address(
+        self, address: Address, require_online_peer: bool = False
+    ) -> Optional[Room]:
         msg = (
             f"address not health checked: "
             f"node: {self._user_id}, "
             f"peer: {to_normalized_address(address)}"
         )
         assert address and self._address_mgr.is_address_known(address), msg
 
+        room_candidates = []
         room_ids = self._get_room_ids_for_address(address)
         if room_ids:  # if we know any room for this user, use the first one
             # This loop is used to ignore any broadcast rooms that may have 'polluted' the
```

Comment:
Just as I read that comment, do we have an issue to track that?

Reply:
The comment is a bit outdated. We should keep that loop to make sure. Also related to #5343.